src/assets/js/lang.js

```diff
@@ -33,6 +34,7 @@ const dict = {
     back: "Voltando ao trabalho:",
     stop: "Parei! 🙂:",
     leftHours: "Horas restantes:",
+    overtime: "Overtime:",
```

Comment:
in portuguese "Overtime" is "Horas extras"
```suggestion
    overtime: "Horas extras:",
```